### ISSUE TITLE
[BUG] Some runs do not show after being imported

### ISSUE THREAD

**Issue by JonatanGeb** [teasit/mftyre-matlab-tool#9]
**Description**
I am having a problem where data from multiple runs do not show up after i try to import them. As of now i have encountered this issue for Formula Student TTC Round 5 runs 30, 32, 33 and 33. Runs 28, 29 and 30, for example, seem to work as intended, at least there are numerous rows labelled e.g. run 29 showing up after import. I am new to matlab and this app, but when opening the data-files separately they all seem to be okay. 

**Reproduce**
1. Open Magic Formula Tire data app
2. Go to Tire Data tab, add desired files from my downloads folder to the top right box
3. Click Run Import
4. Runs 32 - 34 for example do not appear, run 31 does.

**Screenshots**

**Version of MATLAB**
 - Mac (Intel based)
 - Version R2024a

**Comment by teasit** [comment]
Hi, could you please verify that you are using the RunData files in SI units?

**Comment by JonatanGeb** [comment]
Yes, I am using the files in SI units. RawData_DriveBrake_Matlab_SI is the name of the folder containing the runs I mentioned earlier for example.

<img width="1440" alt="Screenshot 2024-05-14 at 09 46 21" src="https://github.com/teasit/magic-formula-tyre-tool/assets/56603923/04853a00-e033-4333-abb4-2b09cae3c0d0">

This is what it looks like when I open the data separately in Matlab, Pressure and Fz values seem to be in Pa and N. (Fz in the range of 200 to 1200)

**Comment by JonatanGeb** [comment]
Hello

Now I tried importing the tire data on a Windows computer to eliminate that source of error and I did some experimenting...

Current setup: 
Windows 10
Matlab R2024a

With version 1.5.1 of the app the problem remained exactly as it was before, only runs 24 and 31 loaded in.
With version 1.5.0 of the app all files (24, 25, 31-34) loaded in correctly. However I got the same problem as issue #4 .
With version 1.5.0 and with the new parser as posted in the solution of issue #4 runs 32-34 are missing again and I got the following:
![Fitter](https://github.com/teasit/magic-formula-tyre-tool/assets/56603923/c2077131-66dd-4ef4-88f2-bce74858e30b)

As I am using data from Round 5 of FSTTC I wonder if some variables have the wrong names and if that is something I manually have to change in the parser? Has the way of documenting the tire data changed? Since the old parser works but not the old fitter, and the new parser only seems to care about some of the data and then the new fitter lacks data to do its job.

**Comment by teasit** [comment]
Hi, you need to use RunData, not RawData.

**Comment by teasit** [comment]
Also make sure the data you import contains data for the mode you want to fit. TTC exports two RunData sets for each tire (with exceptions). To fit combined slip, you need to load both sets.

Hope that helps
Tom
